src/index.ts

```diff
@@ -380,6 +327,85 @@ export class GalleryComponent extends BaseComponent {
     return null;
   }
 
+  private _escapeHtml = (text: string): string => {

```

Comment:
I wonder if this should really be `text: string | number` since as written, it is receiving non-string inputs... which is why we have to do the String cast below.